Aquality.Selenium.Core/src/Aquality.Selenium.Core/Elements/ElementFinder.cs

```diff
@@ -90,7 +83,7 @@ public ReadOnlyCollection<IWebElement> FindElements(By locator, DesiredState des
 
         private void HandleTimeoutException(WebDriverTimeoutException ex, DesiredState desiredState, By locator, List<IWebElement> foundElements)
         {
-            var message = LocalizationManager.GetLocalizedMessage("loc.no.elements.found.in.state", locator.ToString(), desiredState.StateName);
+            var message = $"No elements with locator '{locator.ToString()}' were found in {desiredState.StateName} state";
```

Comment:
I think each message in framework should be localized, lets add some key and add message to json?

Reply:
I beleive it is not necessary to localize Exceptions